api/conf/base.conf

```diff
@@ -27,6 +27,15 @@ apibuilder.sendErrorsTo="mbryzek@alum.mit.edu"
 
 git.version=0.13.0
 
+default-dispatcher {
```

Comment:
Does this have to be in the akka actor namespace? See https://www.playframework.com/documentation/2.6.x/ThreadPools

Reply:
Yes absolutely